schemas/common/contact-lead.schema.json

```diff
@@ -0,0 +1,84 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/contact-lead",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "XDM Contact and Lead common B2B attributes",
+  "type": "object",
+  "description": "Common B2B attributes for contacts and leads.",
+  "definitions": {
+    "contact-lead": {
+      "properties": {

```

Comment:
As note previously - better descriptions

Reply:
Please refer to the first comment